linuxprivchecker.py

```diff
@@ -358,7 +364,7 @@ def writeResults(msg, results):
 print "    Note: Exploits relying on a compile/scripting language not detected on this system are marked with a '**' but should still be tested!"
 print
 
-print "    The following exploits are ranked higher in probability of success because this script detected a related running process, OS, or mounted file system" 
+print "    The following exploits are ranked higher in probability of success because this script detected a related running process, OS, or mounted file system"
```

Comment:
Can the whitespace here be removed? I haven't run it so I'm unsure if it's for formatting but seems like if you're cleaning whitespace...